src/js/themes/base.js

```diff
@@ -1997,6 +1998,12 @@ export const generate = (baseSpacing = 24, scale = 6) => {
         //   color: undefined,
         // },
         // extend: undefined,
+        nextButton: {
+          pad: { horizontal: 'small', vertical: 'xsmall' },
+        },
+        previousButton: {
+          pad: { vertical: 'xsmall', horizontal: 'small' },
+        },
```

Comment:
```suggestion
        nextButton: {
          pad: { horizontal: 'small', vertical: 'xsmall' },
        },
        previousButton: {
          pad: { horizontal: 'small', vertical: 'xsmall' },
        },
```
Super minor comment but makes it a little easier to read

Also wondering if we should consolidate into one object just to avoid adding extra things into the theme?

Reply:
hmm Id be open to one object @taysea objections here?